frame/oracle/src/lib.rs

```diff
@@ -314,6 +314,8 @@ pub mod pallet {
 		/// Too many weighted averages requested
 		DepthTooLarge,
 		ArithmeticError,
+		/// Block interval is less then stale price
```

Comment:
BlockIntervalLength -> BlockIntervalIsLessThenStalePrice?

Reply:
eh, it seems like a lot of words and the comments get passed with the error